smdebug/tensorflow/keras.py

```diff
@@ -538,22 +538,36 @@ def save_gradients_from_logs(self, gradients):
                     g = g.values
                 self._save_tensor_to_file(export_name, g, collections_to_write)
 
+    def _save_model_inputs_and_outputs_helper(self, collection_key, tensors_to_save, prefix):
+        collections_to_write = (
+            {self.get_collection(collection_key)}
+            if self._is_collection_being_saved_for_step(collection_key)
+            else set()
+        )
+        if isinstance(tensors_to_save, (dict, list)):
+            tensors_to_save = nest.flatten(tensors_to_save)
+        else:
+            tensors_to_save = [tensors_to_save]
+        idx = 0
+        for t_value in tensors_to_save:
```

Comment:
nit: Let's use `enumerate` since it was designed for cases like this.

`for idx, t_value in enumerate(tensors_to_save):`

No need to define idx or increment it manually.